packages/components-date/src/FieldDate/FieldDate.tsx

```diff
@@ -38,15 +38,19 @@ import {
 import type { InputDateProps } from '../InputDate'
 import { InputDate } from '../InputDate'
 
-export interface FieldDateProps extends FieldProps, InputDateProps {}
+export interface FieldDateProps
+  extends FieldProps,
+    FloatingLabelFieldProps,
+    InputDateProps {}
 
 const FieldDateComponent = forwardRef(
   (props: FieldDateProps, ref: Ref<HTMLInputElement>) => {
     const validationMessage = useFormContext(props)
     const id = useID(props.id)
     return (
-      <Field
+      <FloatingLabelField
         {...pickFieldProps(props)}
+        hasValue={!!props.defaultValue}
```

Comment:
If you put an initial date value in the controlled example the label will start in the wrong position. I think you can just use the `getHasValue` helper here like in https://github.com/looker-open-source/components/blob/main/packages/components/src/Form/Fields/FieldTextArea/FieldTextArea.tsx#L49
```suggestion
        hasValue={getHasValue(props)}
```

Reply:
`getHasValue` was complaining about `props`  it seems to work fine with `!!props.defaultValue || !!props.value`